packages/mobile/src/helpers/date/get-strict-time-to-event/get-strict-time-to-event.ts

```diff
@@ -0,0 +1,29 @@
+import dayjs from 'dayjs';
+
+const getStrictTimeToEvent = (date: Date) => {
+  const duration = dayjs.duration(dayjs(date).diff(new Date()));
+  const [days, hours, minutes, seconds] = duration
+    .format('D-HH-mm-ss')
+    .split('-');
```

Comment:
`D-HH-mm-ss` - this should be extracted to a separate enum, e.g. DateTimeFormat

Also instead of doing all of these manipulation, take a look at `dayjs` getters

Reply:
changed

Reply:
I need time in a string format, so I cat get 09, 08, 07... instead of 9 ,8 ,7 format and don't write any extra helper for that